setupTests.js

```diff
@@ -1 +1,2 @@
-import '@testing-library/jest-dom';
\ No newline at end of file
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Don't think this eslint-disable comment is necessary.

Reply:
This is needed:
![image](https://github.com/FEC-CLL/FEC/assets/120051705/bcb2cfe0-2600-41bf-a347-81275cc69c84)